examples/rosettacode/rock_paper_scissor.nit

```diff
@@ -0,0 +1,48 @@
+# This file is part of NIT ( http://www.nitlanguage.org )
+# This program is public domain
+# Task: Programming_Task/Rock-paper-scissors
+# SEE: <http://rosettacode.org/wiki/Rock-paper-scissors>
+#
+# author: Istvan SZALAI 03/06/2015
+
+# Return a value to dertermine who win the match.
+# Return 0 for lose, 1 for win, 2 for equality.
+fun win_or_lose(player, computer: String): Int do
+	if player == computer then return 2
+	if player == "rock" and computer == "scissors" then return 1
+	if player == "rock" and computer == "paper" then return 0
+	if player == "paper" and computer == "rock" then return 1
+	if player == "paper" and computer == "scissors" then return 0
+	if player == "scissors" and computer == "paper" then return 1
+	return 0
+end
+
+# Return a String for the player about what the computer player did.
+fun computer_message(str: String): String do
```

Comment:
I'm not sure about the use of this one, a simple string concatenation could very well do the job